clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/config/EurekaProviderConfiguration.groovy

```diff
@@ -70,18 +64,13 @@ class EurekaProviderConfiguration {
   }
 
   private OkHttpClientConfigurationProperties eurekaClientConfig() {
-    Binder binder = new Binder(
-      ConfigurationPropertySources.from(environment.propertySources),
-      new PropertySourcesPlaceholdersResolver(environment),
-      conversionService)
-
     OkHttpClientConfigurationProperties properties =
       new OkHttpClientConfigurationProperties(
         propagateSpinnakerHeaders: false,
         connectTimoutMs: 10000,
         keyStore: null,
         trustStore: null)
-    binder.bind(
+    Binder.get(environment).bind(
```

Comment:
this is an unrelated change to handle multiple conversionServices present in the application context - I will remove it and PR it separately